src/Microsoft.ML.StandardTrainers/Standard/LinearModelParameters.cs

```diff
@@ -138,7 +138,7 @@ private protected virtual bool SaveAsOnnx(OnnxContext ctx, string[] outputs, str
             Host.CheckValue(ctx, nameof(ctx));
             Host.Check(Utils.Size(outputs) >= 1);
             string opType = "LinearRegressor";
-            string scoreVarName = (Utils.Size(outputs) == 2) ? outputs[1] : outputs[0]; // Get Score from PredictedLabel and/or Score columns
+            string scoreVarName = (Utils.Size(outputs) >= 2) ? outputs[1] : outputs[0]; // Get Score from PredictedLabel and/or Score columns
```

Comment:
What cases result in the output size being greater than 2?

Reply:
When a calibrator is used, the probability column is added.  More specifically, the OVA trainer always creates a probability output column, so the output count will be 3. An alternative would be to distinguish between trainers that need the probability column and those that don't inside OVA, but  I think that would be more involved.